apps/settings/js/carrier.js

```diff
@@ -119,6 +119,15 @@ var CarrierSettings = (function() {
       });
     }
 
+    var isThereNoSimCard = Array.prototype.every.call(
```

Comment:
please add some comment to explain what the intention is, such as `//Disable the data connection toggle if no sim card`